File: IamStupendous.Resume/src/IamStupendous.Resume/wwwroot/scripts/app/skill/chart.ts
/**
 * Chart model class
 */
export class Chart {
    labels: string[];
    data: number[];
    type: string;
    options: any;
    colors: any[];

    constructor(labels: string[],
                data: number[],
                type: string,
                options: any,
                colors: any[]) {
        this.labels = labels;
        this.data = data;
        this.type = type;
        this.options = options;
        this.colors = colors;
    }
}

File: IamStupendous.Resume/src/IamStupendous.Resume/Models/Chart.cs
using System.Collections.Generic;

namespace IamStupendous.Resume.Models
{
    public class Chart
    {
        public IList<string> Labels { get; set; }
        public IList<int> Data { get; set; }
        public string Type { get; set; }
        public string Options { get; set; }
        public IList<string> Colors { get; set; }
    }
}
